<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fm_gestaoEstilo</class>
 <widget class="QDialog" name="fm_gestaoEstilo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>730</width>
    <height>447</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Estilo</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton{
background:#248;
color:#fff;
font-size:12pt;
border:none;
border-radius:10px;
}
QPushButton:hover{
	background:#fff;
	color:#248;
	font-size:12pt;
	border:none;
	border-radius:10px;
}
</string>
  </property>
  <widget class="QTableWidget" name="tw_estilo">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>391</width>
     <height>241</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>150</y>
     <width>251</width>
     <height>271</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Nome</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txt_nomeEstilo"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Cor</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txt_corEstilo"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Alcool</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txt_alcoolEstilo"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>Ingrediente</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txt_ingrediente_estilo"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="btn_add">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
background:#248;
color:#fff;
font-size:12pt;
border:none;
border-radius:10px;
}
QPushButton:hover{
	background:#fff;
	color:#248;
	font-size:12pt;
	border:none;
	border-radius:10px;
}
</string>
        </property>
        <property name="text">
         <string>Adicionar</string>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_salvar">
        <property name="styleSheet">
         <string notr="true">QPushButton{
background:#248;
color:#fff;
font-size:12pt;
border:none;
border-radius:10px;
}
QPushButton:hover{
	background:#fff;
	color:#248;
	font-size:12pt;
	border:none;
	border-radius:10px;
}
</string>
        </property>
        <property name="text">
         <string>Salvar</string>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_excluir">
        <property name="styleSheet">
         <string notr="true">QPushButton{
background:#248;
color:#fff;
font-size:12pt;
border:none;
border-radius:10px;
}
QPushButton:hover{
	background:#fff;
	color:#248;
	font-size:12pt;
	border:none;
	border-radius:10px;
}
</string>
        </property>
        <property name="text">
         <string>Excluir</string>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>30</y>
     <width>137</width>
     <height>72</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_10">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_9">
      <item>
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Filtro</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txt_filtro"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="btn_filtrar">
      <property name="text">
       <string>Filtrar</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>83</width>
     <height>132</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_8">
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Filtar por:</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_7">
      <item>
       <widget class="QRadioButton" name="rb_idEstilo">
        <property name="text">
         <string>ID Estilo</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rb_nome">
        <property name="text">
         <string>Nome</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rb_cor">
        <property name="text">
         <string>Cor</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rb_ingrediente">
        <property name="text">
         <string>Ingrediente</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rb_alcool">
        <property name="text">
         <string>Alcool</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
